.agent-skill-chain/scripts/gate-review.sh

```
@@ -9,18 +9,23 @@
-elif command -v agent-skill-chain >/dev/null 2>&1; then
-  CLI=(agent-skill-chain)
-else
-  echo "agent-skill-chain CLI が見つかりません（bin/agents-md.js 未ビルド、node_modules/.bin/agent-skill-chain 不在、PATH上にも無し）。'npm run build' を実行するか agent-skill-chain を導入してください。" >&2
+# >>> agent-skill-chain CLI resolver preamble >>>
+_ASC_WRAPPER_DIR="$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)"
+_ASC_CLI_RESOLVE_PATH="$_ASC_WRAPPER_DIR/../scripts/cli-resolve.sh"
+if [[ ! -r "$_ASC_CLI_RESOLVE_PATH" ]]; then
+  echo "agent-skill-chain CLI の共有実装を解決できません（探索パス: ${_ASC_CLI_RESOLVE_PATH}）。" >&2
+  exit 1
+fi
+if ! source "$_ASC_CLI_RESOLVE_PATH"; then
```

Shellcheck (0.11.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)